index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="top-bar">
+        <a href="#" class="logo">My logo</a>
+        <nav class="menu"><a class="menu-icon" href=""><img src="images/menu.png" alt=""></a></nav> 
+        </header>
```

Comment:
- [ ] Good job for using `header` tag for toolbar section , `a` for log and `nav` tag for hamburger menu. But for hamburger menu you try to import `menu.png` form images folder unfortunately there is no images folder in your Repo, so please try to fix it
![review](https://user-images.githubusercontent.com/93598189/185222152-5032f6b4-259d-4a0d-aaa8-894f422a4d6f.PNG)
.